Description: fix tautological enum compare warnings when built with clang 9
Author: Gianfranco Costamagna <locutusofborg@debian.org>

Forwarded: pending
Last-Update: 2019-12-31

--- a/dlls/gdiplus/graphics.c
+++ b/dlls/gdiplus/graphics.c
@@ -6970,8 +6970,8 @@ GpStatus WINGDIPAPI GdipTransformPoints(
                                         GpCoordinateSpace src_space, GpPointF *points, INT count)
 {
     if(!graphics || !points || count <= 0 ||
-       dst_space < 0 || dst_space > CoordinateSpaceDevice ||
-       src_space < 0 || src_space > CoordinateSpaceDevice)
+       dst_space > CoordinateSpaceDevice ||
+       src_space > CoordinateSpaceDevice)
         return InvalidParameter;
 
     if(graphics->busy)
